set.mm

```diff
@@ -319161,50 +319161,133 @@ can be the countable union of rare closed subsets (where rare means
+      adantll eqcomd reseq1d wss subgss xpss12 resabs1d eqtr4d eleq1d wb cmetss
+      cmscmet mstopn fveq2d eleq2d 3bitrd mpbird iscms sylanbrc ) CGHZCUAHZIZBA
+      HZIZDJHDKLZDMLZWJUBZNZWJUCLZHZDGHWHDCBUDUEZJEWFCJHZWGWOJHWDWPWECUFOZBCAUO
+      UGUHWHWNWJCUILZPLZHZWHBWJWSWHBCUJLZHZBWJQWHCUKHZBCULLZHZXBWFXCWGWEXCWDCUM
+      ROWFCUNHZWGXEWEXFWDCUPRABXDCFXDSZUQUGXDBCXGURUSZBCDEUTTZWEWGBWSHWDABWRCFW
+      RSZVAVEVBWHWNCKLZCMLZXLUBZNZWKNZWMHZWJXNVCLZPLZHZWTWHWLXOWMWHWLXKWKNXOWHW
+      IXKWKWHXKWIWGXKWIQWFBXKCDAEXKSVDRVFVGWHXKWKXMWHWJXLVHZXTWKXMVHWHWJXAHXTWH
+      BWJXAXIXHVBXLWJCXLSZVITZYBWJXLWJXLVJUSVKVLVMWHXNXLUCLHZXPXSVNWFYCWGWDYCWE
+      XNCXLYAXNSZVPOOXNXQXLWJXQSVOTWHXRWSWJWHXQWRPWHWRXQWHWPWRXQQWFWPWGWQOXNWRC
+      XLXJYAYDVQTVFVRVSVTWAWLDWJWJSWLSWBWC $.
+
+    $( Banach space property of a closed subspace: a closed subspace of a
+       Banach space which is also a subcomplex pre-Hilbert space is a Banach
+       space.  Remark: the assumption that the Banach space must be a
+       (subcomplex) pre-Hilbert space is required because the definition of
+       ` ClSubSp ` is based on an inner product.  If ` ClSubSp ` was
+       generalized for arbitrary topological spaces, this assuption could be
+       omitted.  (Proposed by BJ, 7-Oct-2022.)  (Contributed by AV,
+       8-Oct-2022.) $)
+    bncssbn $p |- ( ( ( W e. Ban /\ W e. CPreHil ) /\ U e. S )
+                    -> X e. Ban ) $=
+      ( cbn wcel ccph wa cnvc csca cfv ccms clss bnnvc eqid bnsca jca ad2antrr
+      bncms cmscsscms sylanl1 cphl cphphl adantl csslss sylan cmslssbn syl12anc
+      ) CGHZCIHZJZBAHZJCKHZCLMZNHZJZDNHZBCOMZHZDGHUKURULUNUKUOUQCPUPCUPQRSTUKCN
+      HULUNUSCUAABCDEFUBUCUMCUDHZUNVAULVBUKCUEUFABUTCFUTQZUGUHUTBCDEVCUIUJ $.
+  $}
+
   ${
     cssbn.x $e |- X = ( W |`s U ) $.
     cssbn.s $e |- S = ( LSubSp ` W ) $.
     cssbn.d $e |- D = ( ( dist ` W ) |` ( U X. U ) ) $.
-    $( Banach space property of a closed subspace: a closed subspace of a
-       normed vector space with a closed scalar field is a Banach space.
-       Remark:  In contrast to ` ClSubSp ` , a closed (compact?) subspace is
-       defined by "a linear subspace in which all Cauchy sequences converge to
-       a point in the subspace".  (Contributed by NM, 10-Apr-2008.)  (Revised
-       by AV, 6-Oct-2022.) $)
+    $( Banach space property of a complete subspace: a complete subspace of a
+       normed vector space with a complete scalar field is a Banach space.
+       Remark:  In contrast to ` ClSubSp ` , a complete subspace is defined by
+       "a linear subspace in which all Cauchy sequences converge to a point in
+       the subspace".  (Contributed by NM, 10-Apr-2008.)  (Revised by AV,
+       6-Oct-2022.)  (Proof shortened by AV, 9-Oct-2022.) $)
     cssbn $p |- ( ( ( W e. NrmVec /\ ( Scalar ` W ) e. CMetSp /\ U e. S )
                     /\ ( Cau ` D ) C_ dom ( ~~>t ` ( MetOpen ` D ) ) )
                   -> X e. Ban ) $=
```

Comment:
To me, this theorem is superseded by cmslssbn and can be deprecated.

Reply:
Since this is the translation of the deprecated theorem of part 19, I would like to keep it (at least until the issue with the definition of closed subspaces is solved).

Reply:
Ok. Maybe put something to that effect in the comment, like "kept for compatibility with... but superseded by ~... "